src/app/base/countWeight.js

```diff
@@ -0,0 +1,27 @@
+import { range } from './../constants/constants';
+
+// Count weight category by BMI index
+const countWeight = (container, indexBmi) => {
+    const bmi = container.querySelector('.js-index-bmi');
+    const category = container.querySelector('.js-weight-category');
+
+    bmi.textContent = indexBmi;
+
+    if (indexBmi < 15 || indexBmi > 25) {
+        category.classList.add(range.RANGE_LOW_CLASS);
+        bmi.classList.add(range.RANGE_LOW_CLASS);
+        indexBmi < 15 ? category.textContent = 'To Low Weight!' : category.textContent = 'Too High Weight!';
```

Comment:
Programatically this is not bad, but in a real project, you could not hardcode the text. Either take it from some `data` attribute or usually this would be rendered by BE. Also below the `category.textContent`